packages/headless/src/controllers/insight/facets/category-facet/headless-insight-category-facet.ts

```diff
@@ -21,33 +27,31 @@ import {
 } from '../../../../features/insight-search/insight-search-actions';
 import {InsightEngine} from '../../../../app/insight-engine/insight-engine';
 
-export interface InsightCategoryFacetProps extends CategoryFacetProps {}
-
-export interface InsightCategoryFacet extends CoreCategoryFacet {
-  /**
-   * Provides methods to search the facet's values.
-   */
-  facetSearch: CategoryFacetSearch;
-
-  /**
-   * The state of the `Facet` controller.
-   * */
-  state: InsightCategoryFacetState;
-}
-
-export interface InsightCategoryFacetState extends CoreCategoryFacetState {}
+export type {
+  CategoryFacetValue,
+  CategoryFacetOptions,
+  CategoryFacetSearchOptions,
+  CategoryFacetProps,
+  CategoryFacet,
+  CategoryFacetState,
+  CategoryFacetSearch,
+  CategoryFacetSearchState,
+  CategoryFacetSearchResult,
+  CoreCategoryFacet,
+  CoreCategoryFacetState,
```

Comment:
Don't export those?

Reply:
Why do you export the CoreCategoryFacet? you already re-export the CategoryFacet.
Same for the CoreCategoryFacetState, you already export the CategoryFacetState?

I thought the point was only to export the specific ones here and not the core ones anymore.